inc/saitypes.h

```diff
@@ -855,6 +885,22 @@ typedef enum _sai_acl_bind_point_type_t
 
 } sai_acl_bind_point_type_t;
 
+/**
+ * @brief Attribute data for SAI_ACL_TABLE_ATTR_MATCH_TYPE
+ */
+typedef enum _sai_acl_match_type_t
```

Comment:
Can you please rename the enums as:

*   SAI\_ACL\_TABLE\_MATCH\_TYPE\_XXX

And rename the enum type as:

*   sai\_acl\_table\_match\_type\_t

Reply:
renamed